cli/tools/lint.rs

```diff
@@ -110,9 +123,10 @@ pub async fn lint(flags: Flags, lint_flags: LintFlags) -> Result<(), AnyError> {
       let reporter_kind = lint_options.reporter_kind;
       let reporter_lock = Arc::new(Mutex::new(create_reporter(reporter_kind)));
       let lint_rules = get_config_rules_err_empty(lint_options.rules)?;
-      let r = lint_stdin(lint_rules);
-      let success =
-        handle_lint_result(STDIN_FILE_NAME, r, reporter_lock.clone());
+      let file_path = std::env::current_dir().unwrap().join(STDIN_FILE_NAME);
```

Comment:
You could probably get it from `cli_options.initial_cwd` to avoid th unwrap here